app/Notifications/RequestWithdrawalDeniedNotification.php

```
@@ -0,0 +1,69 @@
+        return [HrmsNotifyUserChannel::class];
+    }
+
+    public function getToken()
+    {
+        return $this->token;
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     */
+    public function toMail(object $notifiable): MailMessage
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)